contracts/ExposedNodeRulesList.sol

```diff
@@ -5,23 +5,23 @@ import "./NodeRulesList.sol";
 
 contract ExposedNodeRulesList is NodeRulesList {
 
-    function _calculateKey(bytes32 _enodeHigh, bytes32 _enodeLow, bytes16 _ip, uint16 _port) public pure returns(uint256) {
-        return calculateKey(_enodeHigh, _enodeLow, _ip, _port);
+    function _calculateKey(string memory _enodeHigh, string memory _ip, uint16 _port) public pure returns(uint256) {
```

Comment:
We should probably rename `_enodeHigh` to `_enode` (applies for all methods in this contract).